lib/default-es5-plugins.js

```diff
@@ -1,15 +1,36 @@
 "use strict";
 
-module.exports = [
-  ['transform-es2015-template-literals', {loose: true}],
-  ['transform-es2015-arrow-functions'],
-  ['transform-es2015-destructuring', {loose: true}],
-  ['transform-es2015-spread', {loose: true}],
-  ['transform-es2015-parameters'],
-  ['transform-es2015-computed-properties', {loose: true}],
-  ['transform-es2015-shorthand-properties'],
-  ['transform-es2015-block-scoping'],
-  ['check-es2015-constants'],
-  ['transform-es2015-classes', {loose: true}],
-  ['transform-proto-to-assign']
-];
+let stripGlimmerUtils = require('babel-plugin-strip-glimmer-utils');
+let debugMacros = require('babel-plugin-debug-macros')['default'];
```

Comment:
Either I'm missing something in my plugins or something, but this is the only way I could get these plugins to work. Something weird is going on because I didn't have to do this in the Ember build.